src/components/FeedInfo.tsx

```diff
@@ -192,9 +197,24 @@ export class FeedInfo extends React.Component<DispatchProps & StateProps, EditFe
         );
     }
 
+    private tryToAddFeedFromClipboard = () => {
+        const isExistingFeed = this.props.feed.feedUrl.length > 0;
+        if (!isExistingFeed) {
+            Clipboard.getString().then(value => {
+                const link = Utils.getLinkFromText(value);
+                if (link != null) {
+                    this.setState({
+                        url: link,
+                    });
+                    this.fetchFeed().then();
```

Comment:
we could use await and mark tryToAddFeedFromClipboard and componentDidMount async

Reply:
👍